sentry_sdk/utils.py

```diff
@@ -1273,6 +1273,35 @@ class ServerlessTimeoutWarning(Exception):  # noqa: N818
     pass
 
 
+class SentryGraphQLClientError(Exception):
+    """Synthetic exception for GraphQL client errors."""
+
+    pass
+
+
+def _get_graphql_operation_name(query):
+    # type: (Dict[str, Any]) -> str
+    if query.get("operationName"):
+        return query["operationName"]
+
+    query = query["query"].strip()
+
+    match = re.match(r"^[a-z]* *([a-zA-Z0-9]+) \(?.*\)? *\{", query)
```

Comment:
Should we make this case insensitive to be on the save side?

Reply:
Changed, also added a small testcase for the two graphql util functions.